lib/src/types/c2_full_results.dart

```diff
@@ -0,0 +1,43 @@
+// ignore_for_file: invalid_annotation_target
+
+part of 'index.dart';
+
+@freezed
+class C2FullResults with _$C2FullResults {
+  //TODO: figure out how to get this into JSON as time_formatted
+  // String get timeFormatted => Duration(
+  //         seconds: this.time.toInt(),
+  //         milliseconds:
+  //             (this.time.remainder(1) * Duration.millisecondsPerSecond).toInt())
+  //     .toString();
+
+  C2FullResults._();
+
+  factory C2FullResults({
+    @JsonKey(name: 'id') @Default(0) int id,
+    @JsonKey(name: 'user_id') @Default(0) int userId,
+    @JsonKey(name: 'date') @TimestampConverter() required DateTime date,
```

Comment:
I notice in all the refactoring the field name for this got renamed from `endDate` (which i find to be more specific description of its purpose). I assume it was changed back to try and get the json keys to work automatically, but figured id check if theres a reason this cant be changed back now that the `@JsonKey` annotation is back.

Reply:
this is ok to change but make crewlab a migration guide for name changes